src/core/env/src/main/scala/NativeLoader.java

```diff
@@ -25,16 +25,15 @@
  * the libraries will be loaded in the order specified in the manifest. The native libraries should be
  * in folders describing the OS they run on: linux, windows, mac. </p>
  * */
-public class NativeLoader {
+public class NativeLoader implements Serializable {
 
     private String resourcesPath;
     private Boolean extractionDone = false;
     private File tempDir;
-    private Object lock = new Object();
 
     public NativeLoader(String topLevelResourcesPath) throws IOException{
         this.resourcesPath = getResourcesPath(topLevelResourcesPath);
-        tempDir = Files.createTempDirectory("tmp").toFile();
+        tempDir = Files.createTempDirectory("mml-natives").toFile();
```

Comment:
Whew -- thanks!!  It'll help when I clear out future messes on the build machine.

Reply:
yeah having that be the same prefix as cntk's natives was a fat 0